arangod/Aql/Query.cpp

```diff
@@ -2674,6 +2705,23 @@ Query::buildSerializeQueryDataCallback(
           }
           return true;
         });
+    // Views separately, if requested:
+    if (flags.includeViewsSeparately) {
+      builder.add(VPackValue("views"));
+      collections().toVelocyPack(
+          builder,
+          /*filter*/ [flags, this](std::string const& name, Collection const&) {
+            if (name.empty()) {
+              return false;
+            }
+            if (!flags.includeNumericIds &&
+                (name.front() >= '0' && name.front() <= '9')) {
+              // exclude numeric collection ids from the serialization.
+              return false;
+            }
+            return this->resolver().getCollection(name) == nullptr;
```

Comment:
Is there maybe a better way to figure out that `name` is a view? Just checking that it's not a collection and concluding it must be a view is probably right, but immediately raises questions when reading the code, and seems like something that might break in the future.

Reply:
Good idea, I will try `getView` in the collection name resolver and see if this is **not** a nullptr. In this particular place we anyway expect these things here to be views, so this cannot hurt... (I think)

Reply:
Seems to work, done.